tests/integration/test_export_and_heudiconv.py

```diff
@@ -159,3 +161,49 @@ def test_heudiconv():
 
     # cleanup output -- for debugging comment this out
     shutil.rmtree(bids_root_dir, ignore_errors=True)
+
+
+def test_unauthorized_user_exception_handling():
```

Comment:
Can you merge the prior PR and then merge `main` so this doesn't show up here?

Reply:
@mirestrepo Can we add a blocking requirement for an updated test under the next feature? I would like to focus on developing a robust test after implementing the IntendedFor acquisition tag key (ace-bold~funcA). Unless you're okay with me completing that work under this feature branch. As for the prior PR / merging to main, I might need to drop by your office for a quick elaboration.